mod-tool/src/main/scala/at/forsyte/apalache/tla/tooling/opt/AbstractCheckerCmd.scala

```diff
@@ -22,11 +21,11 @@ abstract class AbstractCheckerCmd(val name: String, description: String)
         "default: Init")
   var next: Option[String] =
     opt[Option[String]](name = "next", default = None, description = "the name of a transition operator, default: Next")
-  var inv: Option[String] =
-    opt[Option[String]](name = "inv", default = None, description = "the name of an invariant operator, e.g., Inv")
-  var temporal: Option[String] =
-    opt[Option[String]](name = "temporal", default = None,
-        description = "the name of a temporal property, e.g. Property")
+  var inv =
+    opt[Option[List[String]]](name = "inv", default = None,
+        description = "the name of an invariant operator, e.g., Inv")
+  var temporal = opt[Option[List[String]]](name = "temporal", default = None,
+      description = "the name of a temporal property, e.g. Property")
```

Comment:
@konnov Note that this now technically makes it possible for users of the CLI to pass in lists of invariants  or temporal properties. This possibility seemed to be naturally implied by the way we use these internally, but I'm not sure if there's a reason we want to *avoid* exposing this functionality to users. What do you think?

Reply:
Right. I think we always had this possibility internally, e.g., a TLC config allows for multiple invariants. However, we were stuck with the arguments parser, which did not let that behavior.

Reply:
Ok nice! Then I'll update the CLI docs here to communicate this functionality to the users. I'll also update the manual with info on this, but will do that in a followup, since this is big enough :)

Reply:
https://github.com/informalsystems/apalache/issues/2117